wikiconv/README.md

```diff
@@ -23,6 +23,19 @@ We are in the process of uploading the dataset to [Kaggle](https://www.kaggle.co
 
 If you believe there is information in this dataset that should be removed, you can file an github issue or contact `conversationai-questions@google.com`
 
+## Dataset scale
+
+* Here we define conversations as those with at least two unique participants.
```

Comment:
I suggest including using "Conversations" as one heading and then having a separate column for "Conversations with >2 participants" as a separate column. That way numbers match what we have in the paper, and it's clearer.